src/pybind/mgr/dashboard/services/ceph_service.py

```diff
@@ -298,6 +301,10 @@ def get_smart_data_by_daemon(daemon_type, daemon_id):
                 if device['devid'] not in smart_data:
                     smart_data.update(
                         CephService._get_smart_data_by_device(device))
+        else:
+            msg = '[SMART] could not retrieve device list from daemon with type %s and ' +\
+                'with ID %d'
```

Comment:
```suggestion
                'with ID %s'
```

Maybe we can fix this issue in the PR without creating a backport: https://tracker.ceph.com/issues/48245

Reply:
We could also wait until the PR targeting master has been merged and I could cherry-pick the commit into this PR, what do you think?

Reply:
That will make perfect sense to me! Thanks for waiting.